torchtext/csrc/register_bindings.cpp

```diff
@@ -0,0 +1,105 @@
+#include <regex_tokenizer.h>
+#include <sentencepiece.h>
+#include <torch/script.h>
+#include <vectors.h>
+#include <vocab.h>
+
+namespace torchtext {
+
```

Comment:
I think now that the official documentation is updated to use `TORCH_LIBRARY` macro, which makes the registration cleaner and simpler, we can also migrate to this new approach, unless there is some issue about it. Did you try it?

Reply:
Will do this in a follow up PR!